shell/src/main/java/alluxio/cli/bundler/CollectInfo.java

```diff
@@ -60,6 +61,20 @@
  */
 public class CollectInfo extends AbstractShell {
   private static final Logger LOG = LoggerFactory.getLogger(CollectInfo.class);
+  private static final String USAGE =
+          "USAGE: collectInfo [--max-threads <threadNum>] [--local]\n\n"
+                  + "collectInfo runs a set of sub-commands which collect information"
+                  + "about your Alluxio cluster. In the end of the run, "
+                  + "the collected information will be written to files "
+                  + "and bundled into one tarball.\n"
+                  + "[--max-threads <threadNum>] controlls how many threads this command uses. "
+                  + "By default it allocates one thread for each host."
+                  + "Use a smaller number to constrain the network IO when transmitting tarballs.\n"
+                  + "[--local] specifies this command should only collect "
+                  + "information about the localhost.\n"
+                  + "NOTE: You should ALWAYS check the credential risk about this command at\n"
+                  + "https://docs.alluxio.io/os/user/edge/en/operation/Troubleshooting.html"
```

Comment:
Left the link to doc page, do you think the users will follow the link?

Reply:
I think you should explicitly put a message saying "This command MAY bundle credentials. To understand the risks refer to the docs here."

And ideally you can link to a specific subsection of the page not the main page, but that's more a nice to have.

Reply:
Adjusted the message to your advice. Also changed `NOTE` to `WARNING`.